core/arch/arm/kernel/thread_a64.S

```diff
@@ -1312,8 +1305,12 @@ END_FUNC el0_sync_abort
 	/* Restore x0..x1 */
 	load_xregs sp, THREAD_CORE_LOCAL_X0, 0, 1
 
+#if defined(CFG_SECURE_PARTITION)
+	b thread_return_to_sp_interrupt_handler
```

Comment:
We don't guarantee that native interrupts are always masked at S-EL0, they are always unmasked for TAs.